paper/paper.bib

```diff
@@ -0,0 +1,184 @@
+@article{ac2000,
+  author = {A Agresti and B Caffo},
+  year = {2000},
+  title = {Simple and Effective Confidence Intervals for Proportions and Differences of Proportions Result from Adding Two Successes and Two Failures},
+  journal = {The Americal Statistician},
+  volume = {54},
+  number = {4},
+  pages = {280-288},
+  doi = {10.2307/2685779}
+}
+
+@article{barker2002,
+  author = {L Barker},
+  title = {A Comparison of Nine Confidence Intervals for a Poisson Parameter When the Expected Number of Events is ≤ 5},
+  journal = {The Americal Statistician},
+  volume = {56},
+  number = {2},
+  pages = {85-89},
+  year = {2002},
+  doi = {10.1198/000313002317572736}
+  }
+
+  @article{ba1986,
+  author = {J M Bland and D G Altman},
+  title = {Statistical methods for assessing agreement between two methods of clinical measurement },
+  year = {1986},
+  journal = {Lancet},
+  volume = {i(8476)},
+  pages = {307-310},
+  doi = {10.1016/S0140-6736(86)90837-8}
+}
+
+@ARTICLE{bland2007,
+  author = {J M Bland and D G Altman},
+  title = {Agreement between methods of measurement with multiple observations
+	per individual},
```

Comment:
Why line return? Maybe it does not matter.

Reply:
that reference isnt actually used... will remove